kits/mou-drafter/README.md

```
@@ -0,0 +1,135 @@
+
+To support this kit, your Lamatic flow (`mou-drafter`) must contain the following node graph:
+
+```mermaid
+graph TD
+  A[API Request Trigger] --> B[Code Node: Validate Input]
+  B --> C[LLM Node: Clause Generator]
+  C --> D[Code Node: Assemble LaTeX]
+  D --> E[API Response]
+```
+
+### 1. Validate Input Node
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)